src/libs/language-python/api-parser/tests/modules/two_dawson_functions.py

```diff
@@ -0,0 +1,19 @@
+""" A module with two dawson-valid Python functions """
+
+def myfunction(event, context):
+    """ This function is dawson-valid """
+    print(event, context)
+    return "Hello Function"
+
+myfunction.api = {
+    "path": "/test"
```

Comment:
small change needed here: path should not start with a slash (see https://dawson.sh/docs.html#4-function-configuration)